.travis.yml

```diff
@@ -29,11 +29,10 @@ cache:
 
 matrix:
   include:
-    - name: Test Kyuubi w/ Default Profile
+    - name: Test Kyuubi w/ -Pspark-3.0 -Pspark-hadoop-2.7
       env:
-        - PROFILE=""
+        - PROFILE="-Pspark-3.0 -Pspark-hadoop-2.7"
         - JOB="build"
-        - CODECOV="true"
     - name: Test Kyuubi w/ -Pspark-3.0 and Spark 3.1 binary
```

Comment:
this shall be -Pspark-3.1 and Spark 3.0 binary

Reply:
make sense